src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -366,9 +371,31 @@ public void Init(MicrobeEditor editor)
     public override void _Process(float delta)
     {
         // Update mutation points
+        float possibleMutationPoints = editor.MutationPoints - editor.OrganelleCost;
         mutationPointsBar.MaxValue = Constants.BASE_MUTATION_POINTS;
-        mutationPointsBar.Value = editor.MutationPoints;
-        mutationPointsLabel.Text = $"{editor.MutationPoints:F0} / {Constants.BASE_MUTATION_POINTS:F0}";
+        mutationPointsBar.Value = possibleMutationPoints;
+        mutationPointsSubtractBar.MaxValue = Constants.BASE_MUTATION_POINTS;
+        mutationPointsSubtractBar.Value = editor.MutationPoints;
+        if (possibleMutationPoints != editor.MutationPoints)
+        {
+            mutationPointsLabel.Text =
+                $"({editor.MutationPoints:F0} -> {possibleMutationPoints:F0}) / {Constants.BASE_MUTATION_POINTS:F0}";
+        }
+        else
+        {
+            mutationPointsLabel.Text = $"{editor.MutationPoints:F0} / {Constants.BASE_MUTATION_POINTS:F0}";
+        }
+
+        if (possibleMutationPoints < 0)
+        {
+            mutationPointsSubtractBar.TextureProgress_ =
+                GD.Load<Texture>("res://assets/textures/gui/bevel/MpBarInvalid.png");
+        }
+        else
+        {
+            mutationPointsSubtractBar.TextureProgress_ =
+                GD.Load<Texture>("res://assets/textures/gui/bevel/MpBarSubtract.png");
```

Comment:
GD.Load shouldn't be called each frame. Load in the _Ready method and store the object.